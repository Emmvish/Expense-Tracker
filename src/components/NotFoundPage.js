import React from 'react';
import {Link} from 'react-router-dom';

const NotFoundPage = ()=>(
    <div>
      {/* Client side routing is faster than a href type of server routing. */}
      <p>404 Not Found - <Link to="/">Go Home</Link></p>
    </div>
  )

  export default NotFoundPage;