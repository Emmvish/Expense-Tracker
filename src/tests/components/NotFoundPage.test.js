import React from 'react';
import NotFoundPage from '../../components/NotFoundPage'
import {shallow} from 'enzyme';

test('Render NotFoundPage Component', ()=>{
    const wrapper = shallow(<NotFoundPage />)
    expect(wrapper).toMatchSnapshot();
})