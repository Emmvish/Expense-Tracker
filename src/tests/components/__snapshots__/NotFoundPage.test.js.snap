// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NotFoundPage Component 1`] = `
<div>
  <p>
    404 Not Found - 
    <Link
      replace={false}
      to="/"
    >
      Go Home
    </Link>
  </p>
</div>
`;
